### PR TITLE
frontend added components,pages and assets

### PR DESCRIPTION
**Pull request by Manflo27**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <> <Header/>
```
Comment (sjproctor) [inline src/App.js:19]: The Header should be on the next line!

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div className='footer'>
+    <div>©2023 uproar</div>
```
Comment (sjproctor) [inline src/components/Footer.js:6]: All the divs that are nested inside the`<div className='footer'>` should be indented

**Review by sjproctor — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (csaument) [reply src/App.js:19]: Adjusted for next PR.

File: src/components/Footer.js (hunk 2 above)
Comment (csaument) [reply src/components/Footer.js:6]: Adjusted in latest commit to test-coverage.

**Pull request merged**